modules/op-release-notes-1-8.adoc

```diff
@@ -20,6 +20,44 @@ With this update, you can run {pipelines-title} GA 1.8 and later on an OpenShift
 
 include::snippets/technology-preview.adoc[]
 
+=== Adding Git repositories by using webhooks
+
+With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible.
```

Comment:
@sm43 @savitaashture should we point out GitLab and BitBucket support here?

Reply:
Also should we mention the CLI commands that help configuring the webhook as well as web console flow?

Reply:
```suggestion
With this update, as a developer, you can add your Git repository to {pac} by using the webhook mechanism. This way, you can use {pac} even when using GitHub Apps is not feasible. This capability works with GitHub, GitLab, and BitBucket.
```

Reply:
> Also should we mention the CLI commands that help configuring the webhook as well as web console flow?
Typically, release notes don't mention this: it's assumed that we provide interfaces for using a capability. The documentation for the feature should show users both methods for adding repos.